healthtools/config.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+# sites to be scraped
+SITES = {
+    "DOCTORS": "http://medicalboard.co.ke/online-services/retention/?currpage={}",
+    "FOREIGN_DOCTORS": "http://medicalboard.co.ke/online-services/foreign-doctors-license-register/?currpage={}",
+    "CLINICAL_OFFICERS": "http://clinicalofficerscouncil.org/online-services/retention/?currpage={}",
+}
+
+# Doctors document endpoint
+CLOUDSEARCH_DOCTORS_ENDPOINT = {
+    "endpoint_url" : "http://doc-cfa-healthtools-ke-doctors-m34xee6byjmzcgzmovevkjpffy.eu-west-1.cloudsearch.amazonaws.com/",
+    "aws_access_key_id": os.getenv("MORPH_AWS_ACCESS_KEY_ID"),
+    "aws_secret_access_key": os.getenv("MORPH_AWS_SECRET_KEY"),
+    "region_name": os.getenv("MORPH_AWS_REGION")
+}
+
+# Clinical document endpoint
+CLOUDSEARCH_COS_ENDPOINT = {
+    "endpoint_url" : "http://doc-cfa-healthtools-ke-cos-nhxtw3w5goufkzram4er7sciz4.eu-west-1.cloudsearch.amazonaws.com/",
+    "aws_access_key_id": os.getenv("MORPH_AWS_ACCESS_KEY_ID"),
+    "aws_secret_access_key": os.getenv("MORPH_AWS_SECRET_KEY"),
+    "region_name": os.getenv("MORPH_AWS_REGION")
+}
+
+S3_CONFIG = {
+    "aws_access_key_id": os.getenv("MORPH_AWS_ACCESS_KEY_ID"),
+    "aws_secret_access_key": os.getenv("MORPH_AWS_SECRET_KEY"),
+    "region_name": os.getenv("MORPH_AWS_REGION")
+}
+
+TEST_DIR = os.getcwd() + "/healthtools/tests"
```

Comment:
Just to avoid replication, let's have the AWS configs be in this format:
```py
AWS = {
    'config': {
        'access_id': os.getenv('MORPH_AWS_ACCESS_ID'),
        'secret_key': os.getenv('MORPH_AWS_SECRET_KEY'),
        'region': os.getenv('MORPH_AWS_REGION')
    },
    'cloudsearch': {
        'doctors': '...'
        'cos': '...'
    }
}
```